### ISSUE TITLE
No way to disable "Update at Startup" option

### ISSUE THREAD

**Issue by GoogleCodeExporter** [soul4soul/peerblock#76]
```
What steps will reproduce the problem?
1. Open PeerBlock
2. Update UI will auto-statrt whether it's unchecked in the options or not.
3.

What is the expected output? What do you see instead?


What version of PeerBlock are you using? On what operating system? 0.92 
(r86)  32- or 64-bit? 32-bit.


Please provide any additional information below.  Make sure to attach
peerblock.log and/or any screenshots that would help explain your problem.
I'm using XP SP3 and Windows 7 RC1 (v7100)


```

Original issue reported on code.google.com by `pcd...@gmail.com` on 29 Aug 2009 at 3:52

**Comment by GoogleCodeExporter** [comment]
```
This references the "lists" update feature, btw...
```

Original comment by `pcd...@gmail.com` on 29 Aug 2009 at 3:55

**Comment by GoogleCodeExporter** [comment]
```
Yes, we explicitly disabled this option (hardcoding it to true) in recent 
releases,
due to Issue #22 - PeerBlock, like PG2 before it, doesn't actually start 
blocking
until having run through the list-update code.  I would very much like to fix 
Issue
#22, however it's ended up being much more difficult to track down than 
originally
anticipated.  So since not updating at startup means you'll be unprotected, we
decided to not give people that option for now.

Once we fix Issue #22 we'll re-enable the "update at startup" option.  I'll 
leave
this bug open in the meantime to make sure we don't forget.
```

Original comment by `peerbloc...@gmail.com` on 29 Aug 2009 at 5:11
- Changed title: **No way to disable "Update at Startup" option**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
As the underlying bug which necessitated this behavior has now apparently been 
fixed,
we've re-enabled this option in page 2 of PeerBlock's settings.
```

Original comment by `peerbloc...@gmail.com` on 6 Nov 2009 at 6:05
- Changed state: **Fixed**
